src/ray/gcs/gcs_client/accessor.cc

```diff
@@ -1320,13 +1392,15 @@ Status InternalKVAccessor::AsyncInternalKVKeys(
   rpc::InternalKVKeysRequest req;
   req.set_prefix(prefix);
   client_impl_->GetGcsRpcClient().InternalKVKeys(
-      req, [callback](const Status &status, const rpc::InternalKVKeysReply &reply) {
+      req,
```

Comment:
I think all of InternalKV* can be removed and use SYNC_ instead here. I will probably do this in the follow up PR to avoid breaking changes within one PR